ansible_image_validation/validation-playbooks/blacklisted_drivers_validation.yaml

```diff
@@ -17,50 +17,32 @@
     - floppy
     - skx_edac
     - intel_cstate
-  ignore_errors: yes
-
-- name: Check if the drivers are  blacklisted in RHEL 7.9 and 8.3 images
-  shell: modprobe --showconfig | grep blacklist | grep "blacklist {{item}}"
-  register: blacklisted_drivers_specific_rhelversion
-  with_items:
     - amdgpu
   ignore_errors: yes
-  when: ansible_distribution_version == "8.3" or ansible_distribution_version == "7.9"
 
 - name: Initialize an empty list for storing blacklisted drivers
   set_fact:
     missing_drivers_allimages: []
 
-- name: Initialize an empty list for storing blacklisted drivers for specific images
-  set_fact:
-    missing_drivers_specific_rhelversion: []
-
 - name: "Get list of missing drivers to be blacklisted in RHEL 7.9 and 8.3 images"
```

Comment:
Do we need this specific check for 7.9 and 8.3 anymore?

Reply:
the title is incorrect, it's checking drivers for all versions